tests/test_E3_indicator_calculation.py

```diff
@@ -692,13 +693,13 @@ def test_add_degree_of_net_zero_energy():
 
 def test_equation_degree_of_net_zero_energy():
```

Comment:
We could also expand the pytests to cover the expected values of 0, 0<value<1, 1, 1<value.

Reply:
done in 30a2475